src/Cache/Cache.php

```diff
@@ -486,6 +571,35 @@ public static function deleteMany($keys, $config = 'default')
         return $engine->deleteMany($keys);
     }
 
+    /**
+     * Delete many keys from the cache.
+     *
+     * ### Usage:
+     *
+     * Deleting multiple keys from the active cache configuration.
+     *
+     * ```
+     * Cache::deleteMultiple(['my_data_1', 'my_data_2']);
+     * ```
+     *
+     * Deleting from a specific cache configuration.
+     *
+     * ```
+     * Cache::deleteMultiple(['my_data_1', 'my_data_2], 'long_term');
+     * ```
+     *
+     * @param array $keys Array of cache keys to be deleted
+     * @param string $config name of the configuration to use. Defaults to 'default'
+     * @return bool
+     * couldn't be removed
```

Comment:
What does this comment mean? Should this be above the params instead with a more clear message?